File: Serene3/Serene3.Web/Modules/com/cetogery/cetogeryColumns.cs
using Serenity.ComponentModel;
using System.ComponentModel;

namespace Serene3.com.Columns;

[ColumnsScript("com.cetogery")]
[BasedOnRow(typeof(cetogeryRow), CheckNames = true)]
public class cetogeryColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int Id { get; set; }
    [EditLink]
    public string Name { get; set; }
    public string Photos { get; set; }
    public string Description { get; set; }
}

File: Serene3/Serene3.Web/Modules/ServerTypes/com/cetogeryColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { cetogeryRow } from "./cetogeryRow";

export interface cetogeryColumns {
    Id: Column<cetogeryRow>;
    Name: Column<cetogeryRow>;
    Photos: Column<cetogeryRow>;
    Description: Column<cetogeryRow>;
}

export class cetogeryColumns extends ColumnsBase<cetogeryRow> {
    static readonly columnsKey = 'com.cetogery';
    static readonly Fields = fieldsProxy<cetogeryColumns>();
}